apps/damap-frontend/project.json

```diff
@@ -45,8 +45,8 @@
             },
             {
               "type": "anyComponentStyle",
-              "maximumWarning": "4kb",
-              "maximumError": "5kb"
+              "maximumWarning": "5.12kb",
+              "maximumError": "5.13kb"
```

Comment:
minor: instead of adapting these limits, maybe it makes sense to try and reduce component size by splitting it into several components. Additionally, we could check if really all the css is required. Your call.

Reply:
I did try that without changing the logic, therefore i did increase this at the minimum limit so that in the future i am aware of this.